src/config.h

```diff
@@ -203,8 +203,7 @@ class Config {
     }
   }
 
-private:
```

Comment:
Why this removal?

Reply:
There's a private 2 functions up already. Caught by https://clang.llvm.org/extra/clang-tidy/checks/readability/redundant-access-specifiers.html

Reply:
Oooohhh, nice!